### PR TITLE
add support for routes in gateway sal

### PR DESCRIPTION
**Pull request by zaibon**
required for https://github.com/threefoldtech/0-templates/issues/81

### CODE REVIEW

File: JumpscaleLib/sal_zos/gateway/gateway.py [hunk 1]
```diff
@@ -466,6 +532,20 @@ def configure_fw(self):
...
 
+    def configure_routes(self):
+        """
+        Configure routing table
+        """
+        if self.container is None:
+            raise RuntimeError('Can not configure fw when gateway is not deployed')
+
+        for route in self.routes:
+            self.container.client.ip.route.add(
```
Comment (grimpy) [inline JumpscaleLib/sal_zos/gateway/gateway.py]: don't you need to check here that the routes already exist or not?

File: JumpscaleLib/sal_zos/gateway/gateway.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import yaml
 from jumpscale import j
-from .http import HTTPServer
+
+from ..abstracts import Collection
+from .cloudinit import CloudInit
 from .dhcp import DHCP
 from .firewall import Firewall
-from ..abstracts import Collection
+from .http import HTTPServer
```
Comment (grimpy) [inline JumpscaleLib/sal_zos/gateway/gateway.py]: why is this import required?

Comment (zaibon) [reply JumpscaleLib/sal_zos/gateway/gateway.py]: It as already there, and it's used line 519

File: JumpscaleLib/sal_zos/gateway/gateway.py (hunk 1 above)
Comment (zaibon) [reply JumpscaleLib/sal_zos/gateway/gateway.py]: indeed
